ci/builders/local_engine.json

```diff
@@ -19,6 +19,26 @@
         "targets": []
       }
     },
+    {
+      "drone_dimensions": [
```

Comment:
I hope to get the builds in this file running on CI soon, so wondering if local workflows will need both the unopt and non-unopt version of android_debug_arm64?

Reply:
We talked offline, local builds will _probably_ all be `unopt` (the other configurations will be covered by release builders), so I am going to remove `android_debug_arm64` (/cc @johnmccutchan) from this file, and then continue to chat with John about "what should we do for duplicate names in general".

Reply:
I only use android_debug_arm64 and never the _unopt_ variant. So please don't remove it.

Reply:
> I only use android_debug_arm64 and never the _unopt_ variant. So please don't remove it.

Ack. Can I clarify though, why is [this builder](https://github.com/flutter/engine/blob/622b37282f20e094cf43ff9f7485305199840121/ci/builders/linux_android_debug_engine.json#L118) not sufficient?

Reply:
Chatted offline, because that builder doesn't run/allow `macos`. Reverted that `-` block.